k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml

```diff
@@ -2,7 +2,10 @@
 src-registry: gcr.io/k8s-staging-cluster-api
 registries:
 - name: gcr.io/k8s-staging-cluster-api
-  service-account: service-account@kubernetes-public.iam.gserviceaccount.com
-- name: gcr.io/k8s-gcr-prod
-  service-account: service-account@kubernetes-public.iam.gserviceaccount.com
+- name: us.gcr.io/k8s-gcr-prod
```

Comment:
If we wanted this have subdirs, would it work?  (we don't yet, but might for things like e2e test images)

Reply:
Do you mean this: https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/20 ?
We need to implement that feature...

To rephrase, that issue is describing this scenario:

Push to `gcr.io/k8s-staging-cluster-api/foo-image`, but we want it to show up as either

(1) `gcr.io/k8s-gcr-prod/bar-image` (notice the image rename)
(2) `gcr.io/k8s-gcr-prod/some/different/subdir/path/foo-image` (notice how the "directory path" to the image is different)

or a mix of both (1) and (2) (image rename *and* path rename).

This is what you mean, right?

Reply:
Mostly I care about (2), I think.

Reply:
Ok, I will start working on (2) and report progress in kubernetes-sigs/k8s-container-image-promoter#20.

Reply:
I created a new PR to handle subdirs (renames) https://github.com/kubernetes-sigs/k8s-container-image-promoter/pull/40. It supports the general case of renaming both the image name itself or the subdirectory (or both). Rename handling required some refactoring and a slight tweak to the promotion logic, which resulted in some delay. Anyway after that's merged, I have to

- tag the new promoter release
- create new promoter images
- update the Prowjobs to use the new promoter images
- update this PR (the redundant `src-registry` toplevel field is no longer required)

It's mostly housekeeping stuff at this point. The e2e test suite is the next priority item...